src/theory/bv/theory_bv_utils.cpp

```diff
@@ -26,6 +26,17 @@ namespace theory {
 namespace bv {
 namespace utils {
 
+Node mkSum(std::vector<Node>& children, unsigned width)
+{
+  std::size_t nchildren = children.size();
+
+  if (nchildren == 0)
+    return bv::utils::mkZero(width);
```

Comment:
You can use mkZero without the scopes here (since this is in namespace bv::utils).